# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../kdebase-runtime/kdebase-runtime-4.6.3-no-install-hicolor-index.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- kdebase-runtime-4.6.3/pics/hicolor/CMakeLists.txt.orig	2011-05-26 16:30:54.800003715 +0200
+++ kdebase-runtime-4.6.3/pics/hicolor/CMakeLists.txt	2011-05-26 16:31:04.888003715 +0200
@@ -1,5 +1,5 @@
 
 ########### install files ###############
 
-install( FILES index.theme  DESTINATION  ${ICON_INSTALL_DIR}/hicolor )
+# install( FILES index.theme  DESTINATION  ${ICON_INSTALL_DIR}/hicolor )
 
